src/templates/sidebar.html

```diff
@@ -1,302 +1,525 @@
-                              <input type="text" class="search-query rounded" placeholder="Search" name="q">
-                            </div>
-                          </form>
-                      </div>
-                      <li class="Divider">GraphQL Reference</li>
-                      <li><a href="/graphql/reference">Reference</a></li>
-
-                      <li class="Divider">GraphQL Tutorials</li>
-                      <li><a href="/graphql/custom-widget">Custom Widget via GraphQL</a></li>
-                  </ul>
-              </li>
-
-                <li class="Divider">Mobile Resources</li>
-                        <li><a href="/mobile/branch-metrics/">Branch Metrics Integration - Quickstart</a></li>
-                        <li><a href="/mobile/branch-metrics/reference/">Branch Metrics Reference</a></li>
-                    </ul>
+                <li>
+                  <a href="/segment/">Segment Integration Introduction</a>
                 </li>
+                <li>
+                  <a href="/integrations/segment-v2/">Segment Integration V2</a>
+                </li>
+                <li>
+                  <a href="/integrations/segment-v2/subscription/"
+                    >Segment Subscription</a
+                  >
+                </li>
+                <li>
+                  <a href="/integrations/segment-v2/stream/">Segment Stream</a>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="/api/"><i class="fa fa-fw fa-desktop"></i> REST API</a>
+          <ul>
+            <div class="main-site-logo">
+              <a href="/">
+                <img src="/assets/images/saasquatch-logo.png" />
+              </a>
+            </div>
+            <div class="help-center-logo">
+              <a href="/">
+                <img src="/assets/images/helpcenter.png" />
+              </a>
+            </div>
+            <div class="docs-searchbox-full">
+              <form class="js-search-form form-search" action="/search/">
+                <div class="input-append">
+                  <input
+                    type="text"
+                    class="search-query rounded"
+                    placeholder="Search"
+                    name="q"
+                  />
+                </div>
+              </form>
+            </div>
+            <li>
+              <a class="back-link" href="#mm-5">
+                <i class="fa fa-chevron-left"></i> Go Back
+              </a>
+            </li>
+            <li><a href="/api/authentication">Authentication</a></li>
+            <li><a href="/api/openendpoints">API Open Endpoints</a></li>
+            <li><a href="/api/errors">Errors</a></li>
+            <li><a href="/api/webhooks">Webhooks</a></li>
 
+            <li class="Divider">API Reference</li>
+            <li><a href="/api/methods">Full list of Methods</a></li>
 
+            <APILIST />
```

Comment:
The space breaks this.

```suggestion
            <APILIST/>
```

Reply:
This seems to have been from Prettier's auto-formatting. Will have to use a comment just to make sure it ignores this line